### DISCUSSION TITLE
Question: How do I enter an "=" equals sign?

### DISCUSSION THREAD

**Discussion by MatthewRBlair** [shorepine/tulipcc#146] [General]
Either I am dumb or it isn't supported, please tell me which. To be clear, this is for the tdeck.

**Comment by bwhitman** [comment]
You're not dumb! 

The T-Deck doesn't have an `=` symbol on its keyboard. 

We will have to find a symbol on the T-Deck's keyboard that is less used in Python/Tulip. And that symbol can emit a `=`. 

You would do that around this line: https://github.com/bwhitman/tulipcc/blob/34f7a782343b1c1a156522c4640f581aae914910/tulip/tdeck/tdeck_keyboard.c#L101

As far as a key, maybe `$` or `;` or `!`? I'm trying to think of something not too critical in Python. You can see the list of keys that the T-Deck emits here: https://github.com/Xinyuan-LilyGO/T-Deck/blob/master/examples/Keyboard_ESP32C3/Keyboard_ESP32C3.ino

**Comment by hreikin** [comment]
@MatthewRBlair You should be able to get an `=` symbol by pressing the `sym` key and then holding `shift` while pressing `n`/`,`. This works because the keyboard firmware simply minuses 32 from the characters position whenever shift is pressed, with `sym` enabled the `,` then becomes `=`.

@bwhitman I have a PR i have been working on to expand the keyboards functionality and provide many of the missing keys/characters, once i have cleaned it up later i will open a PR with the proposed changes.

**Comment by bwhitman** [comment]
@MatthewRBlair with #147 now committed, you can type an equals sign! (And many other things). Thanks @hreikin !
